Dockerfile

```diff
@@ -1,3 +1,4 @@
+# Build Geth in a stock Go builder container
 FROM golang:1.15-alpine as builder
 
 RUN apk add --no-cache make gcc musl-dev linux-headers git bash
```

Comment:
This docker file is written when the golang 1.15 base image is not ready,, please refer to https://github.com/ethereum/go-ethereum/blob/master/Dockerfile, to use golang 1.16